hw/xbox/nv2a/nv2a_pgraph.c

```diff
@@ -4080,6 +4127,116 @@ static uint8_t* convert_texture_data(const TextureShape s,
                                      unsigned int row_pitch,
                                      unsigned int slice_pitch)
 {
+    //FIXME: Handle all formats
+    if (true) {
+
+      // Get texture format information
+      ColorFormatInfo f = kelvin_color_format_map[s.color_format];
+
+      //FIXME: Extend for other formats in the main texture table
+      printf("Converting format: 0x%X\n", s.color_format);
+      const uint32_t* gl_xor_mask_parts = get_gl_sign_bits(f.gl_format, f.gl_type);
+
+      // Collect XOR mask bits
+      uint32_t gl_xor_mask = 0;
+      bool gl_component_used[4] = { false, false, false, false };
+      bool gl_component_signed[4] = { false, false, false, false };
+      for(int i = 0; i < 4; i++) {
+
+        // Use GL swizzle mask to figure out which GL component to use
```

Comment:
I don't remember what I meant by this and the comment doesn't help.
I should improve this comment.